concrete/blocks/express_form/controller.php

```diff
@@ -173,7 +173,7 @@ public function delete()
             // Important – are other blocks in the system using this form? If so, we don't want to delete it!
             $db = $entityManager->getConnection();
             $r = $db->fetchColumn('select count(bID) from btExpressForm where bID <> ? and exFormID = ?', [$this->bID, $this->exFormID]);
-            if ($r == 0) {
+            if (0 == $r) {
```

Comment:
*please* *please* *please* don't use Yoda style 😢

Reply:
Please Visit branch develop concrete5 you will find that  instruction is not attached to ower changment ;-)

Reply:
Yep, that Yoda style was applied in https://github.com/concrete5/concrete5/commit/27d516b3af8f9b39ea1582ee115e81aee2cc981d?w=1#diff-f120e1e91dcc638a79064a5d7ebdcab5R123 (it's another pull request, but the author is still you :wink:)